lib/content-services/src/lib/breadcrumb/breadcrumb.component.scss

```diff
@@ -79,19 +78,18 @@
         font-weight: 600;
         letter-spacing: -0.2px;
         text-align: left;
-        color: var(--adf-theme-foreground-text-color-054);
         flex: 0 1 auto;
         min-width: 35px;
         margin-top: auto;
         text-overflow: ellipsis;
 
         &:hover,
         &.adf-active {
-            color: var(--adf-theme-foreground-text-color-064);
+            color: var(--adf-theme-foreground-text-light-opacity);
```

Comment:
I am not sure about it but opacity 0.64 was replaced by 0.4. If it was based on impression that 0.4 looks better I think it is ok.